P5/Source/Specs/annotation.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" ident="annotation" mode="add" module="linking"
+  xmlns:sch="http://purl.oclc.org/dsdl/schematron">
+  <desc versionDate="2020-07-31" xml:lang="en">An annotation following the <ref target="#WADM">Web
```

Comment:
usually the desc starts with a verb, contains?

Reply:
👍. Fixed. I used "encodes" as the verb, since `<annotation>` is and annotation rather than containing one.

Reply:
I think the verb “represents” would be a better choice. It is certainly well attested in the _Guidelines_. (“Provides”, “contains”, “marks”, “supplies”, “defines”, and “specifies” are other well attested possibilities.)